src/crystalla/blas_helper.cr

```diff
@@ -1,5 +1,5 @@
 module Crystalla::BlasHelper
-  def blas_multiply(other)
+  def blas_multiply_matrix(other)
```

Comment:
Instead of renaming we could take advantage of Crystal's dispatching capabilities. Just replace `blas_multiply_matrix(other)` and `blas_multiply_array(other)` with `blas_multiply(other: Matrix)` and `blas_multiply(other: NdArray)` and that's it! Any algorithm that relies on `blas_multiply` is now agnostic :).